homeassistant/components/fritz/common.py

```diff
@@ -184,9 +186,10 @@ def __init__(
         self,
         hass: HomeAssistant,
         password: str,
+        port: int | None,
```

Comment:
port should never be `None`, since we take it from the configuration entry, where it is a mandatory paramater
```suggestion
        port: int,
```